modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -18,10 +18,16 @@ file. Your cluster can have both control plane and compute machines running on b
 
 Bare-metal compute machines are not supported on clusters that use Kuryr.
 
+[WARNING]
+====
+Bare metal workers are not functional on clusters that run on {rh-openstack} versions after 16.1.6 and before 16.2.4 due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[known issue].
+
+On {rh-openstack} 16.2, clusters that use bare metal workers and the OpenShift SDN can fail depending on the MTU configuration that is described in a link:https://bugzilla.redhat.com/show_bug.cgi?id=2153801[known issue].

```

Comment:
To be clear, we try to avoid linking to live bugs in the docs, as maintenance gets confusing. Maybe we could link those bugs to tickets to edit/remove the text in this PR if they're resolved? 

lgtm, otherwise. Could consider:

```suggestion
Bare metal workers are not functional on clusters that run on {rh-openstack} versions between 16.1.6 and 16.2.4 due to a link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[known issue].

On {rh-openstack} 16.2, clusters that use bare metal workers and the OpenShift SDN can fail depending on the MTU configuration that is described in a link:https://bugzilla.redhat.com/show_bug.cgi?id=2153801[known issue].

```

Reply:
ah okay, I will add that. Thanks for the hint

Reply:
I just added at https://bugzilla.redhat.com/show_bug.cgi?id=2177155